pkg/util/azureclient/helpers.go

```diff
@@ -34,3 +39,47 @@ func ExternalToSdk(in *v20180930preview.OpenShiftManagedCluster) (out *sdk.OpenS
 
 	return
 }
+
+// ResourceGroupFromManagedResourceID returns the resource group name from  a managed resource group identifier. The managed resource
+// group id for managed applications is formatted as follows `/subscriptions/(.+)/resourceGroups/(.+)`
+func ResourceGroupFromManagedResourceID(resourceID string) (string, error) {
```

Comment:
Move all these helpers inside `test/e2erp`. I'd prefer that and moving them back to this package if we ever need to use in prod code vs dumping everything here.

Reply:
done